Makefile

```diff
@@ -71,33 +71,23 @@ update-venv:
 install-pre-commit: update-venv ## install the pre-commit hooks
 	venv/bin/pre-commit install
 
-%.pickle: %.yaml
-	venv/bin/python data_generation/convert_yaml_to_pickle.py $< $@
-
-pickles: climate_categories/data/RCMIP.pickle
-pickles: climate_categories/data/GCB.pickle
-pickles: climate_categories/data/IPCC2006.pickle
-pickles: climate_categories/data/IPCC2006_PRIMAP.pickle
-pickles: climate_categories/data/IPCC1996.pickle
-pickles: climate_categories/data/CRF1999.pickle
-pickles: climate_categories/data/gas.pickle
-pickles: climate_categories/data/CRFDI.pickle
-pickles: climate_categories/data/CRFDI_class.pickle
-pickles: climate_categories/data/BURDI.pickle
-pickles: climate_categories/data/BURDI_class.pickle ## re-generate pickles from yamls
-
-
-climate_categories/data/RCMIP.yaml: data_generation/RCMIP.py data_generation/utils.py
-	venv/bin/python data_generation/RCMIP.py
-
-climate_categories/data/IPCC2006.yaml: data_generation/IPCC2006.py data_generation/utils.py
-	venv/bin/python data_generation/IPCC2006.py
-
-climate_categories/data/IPCC1996.yaml: data_generation/IPCC1996.py data_generation/utils.py
-	venv/bin/python data_generation/IPCC1996.py
```

Comment:
I thought they were covered by the general rule now, will check again.

Reply:
The difference is that the specific rules also have a dependency on `utils.py`, so that the yaml is considered to be outdated if `utils.py` was changed. Maybe not very pressing anymore since `utils.py` is rather static, I added the dependency when I was changing `utils.py` a lot to remove code duplications from `IPCC*.py`.

Reply:
Good point, it's the same problem with `to_python` when i added the comment :-)

Should i add the tasks back?

Reply:
Maybe just add `to_python` and `utils.py` to the general tasks? Worst case, a few more regenerations are being run when utils.py is changed, but that's not so bad.

Reply:
Done in https://github.com/pik-primap/climate_categories/pull/49/commits/02c457bcf1e550c80de10f53a52fba88fcc8d4a2